pimd/pim_zebra.c

```diff
@@ -449,7 +449,11 @@ void pim_zebra_init(void)
 	zclient->zebra_connected = pim_zebra_connected;
 	zclient->nexthop_update = pim_nexthop_update;
 
+#if PIM_IPV == 6
+	zclient_init(zclient, ZEBRA_ROUTE_PIM6, 0, &pimd_privs);
```

Comment:
Can you move this to `pimd/pim_addr.h` please, where all the other `#ifdef` for constants are (call it `ZEBRA_ROUTE_PIM_AF` or something, #define to the correct ZEBRA_ROUTE_*)

Reply:
@eqvinox Addressed in force pushed changes.